datasketch/weighted_minhash.py

```diff
@@ -135,3 +141,86 @@ def minhash(self, v):
             hashvalues[i][0], hashvalues[i][1] = k, int(t[k])
         return WeightedMinHash(self.seed, hashvalues)
 
+    def minhash_many(self, X : Union[sp.sparse.csr_matrix, np.ndarray]) \
```

Comment:
This should have been sp.sparse.spmatrix, which is a parent class of csr_matrix.